db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/jdbc/MssqlJdbcCustomization.java

```diff
@@ -44,6 +51,27 @@ public Instant getInstant(ResultSet rs, String columnName) throws SQLException {
         return Optional.ofNullable(rs.getTimestamp(columnName)).map(Timestamp::toInstant).orElse(null);
     }
 
+    @Override
+    public byte[] getBytes(ResultSet rs, String columnName) throws SQLException {
+        byte[] result;
+        if (driverName.contains("jTDS")) {
+            try (Reader reader = ((Clob) rs.getObject(columnName)).getCharacterStream()) {
+                char[] charArray = new char[8 * 1024];
+                StringBuilder builder = new StringBuilder();
+                int numCharsRead;
+                while ((numCharsRead = reader.read(charArray, 0, charArray.length)) != -1) {
+                    builder.append(charArray, 0, numCharsRead);
+                }
+                result = builder.toString().getBytes(StandardCharsets.UTF_16LE);
+            } catch (IOException e) {
+                throw new SQLException(e);
+            }
+        } else {
+            result = rs.getBytes(columnName);
+        }
```

Comment:
How about making this into a separate JdbcCustomization, e.g. `MssqlJTDSJdbcCustomization`?

Reply:
I created new JdbcCustomization